test/zdtm.py

```diff
@@ -1032,6 +1039,12 @@ def run(action,
 
         return ret
 
+    @staticmethod
+    def exit_signal(ret):
+        if ret == -2:
+            return True
+        return False
```

Comment:
This could be simplified to:
```Python
def exit_signal(ret):
    return ret == -2
```

Reply:
Should've written that in the first place. I will fix it. Thank You :)

If I understand your question correctly,
The run static method returns -1 if criu dump fails.
I changed the run method to return -2 if rpc sends an invalid response (CRIU crashed?).

exit_signal function checks for a crash. So, if ret == -2 then it means CRIU crashed.